scout/server/blueprints/variant/templates/variant/buttons.html

```diff
@@ -119,3 +119,17 @@
     </div>
   </form>
 {% endmacro %}
+
+{% macro reviewer_button(case, variant, case_groups, institute_id) %}
+  {% if config.SCOUT_REVIEWER_URL %}
```

Comment:
Yes; ready as soon as this is merged: https://github.com/Clinical-Genomics/servers/pull/867

Reply:
Why don't you split that PR into 2 PRs (one for stage and one for prod), so stage is testable before prod?

Reply:
This is perhaps more for that PR, but it would still be very testable on stage anyway by deploying the PR on cg-vm1. That would not touch the cg-prod-services server, which would still be on its servers master.

Reply:
Also, I kind of did that with deploying the SRS on stage only before, but it creates a bit of a bad habit: you test with your first PR to change stage, then you make an "untested" PR for merging to the master later. Its nice to have them together. But would either work, sure! 🤷

Reply:
> That would not touch the cg-prod-services server

Somebody could still deploy it on cg-prod-services while deploying their own branch, if they merge after you.. Just saying..

Reply:
Hehe, right, hence no merge on servers until the feature is merged in Scout. To be fair, it would not really do anything right now anyway, just sit in the config.